### PR TITLE
Add tx hash at the top of the send command

### PR DESCRIPTION
**Pull request by TomMD**
### What

Make `send` emit the transaction hash as well.

### Why

Because it is a normal sequence of operations to view the hash during transmission.

### CODE REVIEW

File: cmd/soroban-cli/src/commands/tx/send.rs [hunk 1]
```diff
@@ -56,6 +57,16 @@ impl NetworkRunnable for Cmd {
         };
         let client = network.rpc_client()?;
         let tx_env = super::xdr::tx_envelope_from_stdin()?;
+        match super::xdr::unwrap_envelope_v1(tx_env.clone())
+            .map(|tx| transaction_hash(&tx, &network.network_passphrase))
+        {
+            Ok(Ok(hash)) => {
+                if !globals.map_or(false, |g| g.quiet) {
+                    println!("Transaction Hash: {}", hex::encode(hash));
+                }
```
Comment (Ifropc) [inline cmd/soroban-cli/src/commands/tx/send.rs:66]:
```suggestion
                let print = Print::new(global_args.quiet);
                print.infoln(format!("Transaction Hash: {}", hex::encode(hash)));
```

**Review by Ifropc — COMMENTED** [review]
You can technically use `stellar tx hash` but I'm all in adding more informative output for users
This change LGTM but would like to hear @fnando @leighmcculloch opinion on it

**Review by Ifropc — APPROVED** [review]

File: cmd/soroban-cli/src/commands/tx/send.rs [hunk 2]
```diff
@@ -56,6 +57,16 @@ impl NetworkRunnable for Cmd {
         };
         let client = network.rpc_client()?;
         let tx_env = super::xdr::tx_envelope_from_stdin()?;
+        match super::xdr::unwrap_envelope_v1(tx_env.clone())
+            .map(|tx| transaction_hash(&tx, &network.network_passphrase))
+        {
+            Ok(Ok(hash)) => {
+                if !globals.map_or(false, |g| g.quiet) {
```
Comment (leighmcculloch) [inline cmd/soroban-cli/src/commands/tx/send.rs:64]:
```suggestion
                if !globals.is_some_and(|g| g.quiet) {
```

File: cmd/soroban-cli/src/commands/tx/send.rs [hunk 3]
```diff
@@ -56,6 +57,16 @@ impl NetworkRunnable for Cmd {
...
         let client = network.rpc_client()?;
         let tx_env = super::xdr::tx_envelope_from_stdin()?;
+        match super::xdr::unwrap_envelope_v1(tx_env.clone())
+            .map(|tx| transaction_hash(&tx, &network.network_passphrase))
+        {
+            Ok(Ok(hash)) => {
+                if !globals.is_some_and(|g| g.quiet) {
+                    println!("Transaction Hash: {}", hex::encode(hash));
+                }
+            }
```
Comment (fnando) [inline cmd/soroban-cli/src/commands/tx/send.rs:67]: I think this should be using the Print pattern.

Comment (fnando) [reply cmd/soroban-cli/src/commands/tx/send.rs:67]: oh, looks like @Ifropc already requested the same change.

**Pull request merged**